README.md

```diff
@@ -53,4 +53,11 @@ jobs:
         with:
           name: patch
           path: '*.patch'
+      - name: Upload explanations file as artifact
```

Comment:
Update to use the path. (We are still outputting the summary of the additions and subtractions with the table, so that could still be used to send a comment if the expected diff is always going to be too big.)